.gitignore

```diff
@@ -26,6 +26,7 @@ target
 .eggs
 .egg-info
 *.so
+venv/
```

Comment:
We are using mostly pyenv virtualenvs just as a comment

Reply:
Ah nice! I am already using pyenv but I didn't know this feature 👍🏼 
Still, it doesn't harm to leave `venv` here I think